packages/template-retail-react-app/app/pages/product-list/partials/page-header.jsx

```diff
@@ -26,12 +26,7 @@ const PageHeader = ({category, productSearchResult, isLoading, searchQuery, ...o
                     {`${category?.name || searchQuery || ''}`}
                 </Heading>
                 <Heading as="h2" size="lg" marginRight={2}>
-                    {isServer ? (
-                        <Fragment>({productSearchResult?.total})</Fragment>
-                    ) : (
-                        // Fade in the total when available. When it's changed or not available yet, do not render it
-                        !isLoading && <Fade in={true}>({productSearchResult?.total})</Fade>
-                    )}
+                    {!isLoading && <Fade in={true}>({productSearchResult?.total})</Fade>}
```

Comment:
We have to keep consistency in both server and client or React will prompt an error with hydration
![image](https://user-images.githubusercontent.com/52219283/236285169-7c87f3ed-3b57-4cca-9384-d911367a7940.png)